deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -42,9 +42,6 @@ class NewChatViewController: UITableViewController {
         return searchController.searchBar.text?.isEmpty ?? true
     }
 
-    var syncObserver: Any?
-    var hud: ProgressHud?
```

Comment:
even if the handler is triggered for some reason, this `hud` seems to be never set.